internal/kconfig/kconfig.go

```diff
@@ -58,7 +58,7 @@ func Parse(source io.ReaderAt) (map[string]string, error) {
 	for s.Scan() {
 
 		line := s.Text()
-		err = processKconfigLine(line, ret)
+		err = processKconfigLine(line, ret, filteredConfigs)
```

Comment:
The result map should be pre-allocated to the number of configs we are interested in:

```
ret := make(map[string]string, len(filteredConfigs))
```

This is because in the benchmark we spend a lot of time growing / copying the map.